gui/v5/backend.py

```
@@ -0,0 +1,645 @@
+        # Mock function to simulate sending keep-alive signal
+        self.flight_log.insert(0, "Keep alive signal sent.")
+        self.flightLogUpdated.emit(self.flight_log)
+
+    @Slot(str)
+    def getDroneAction(self, action):
+        if action == 'connect':
+            try:
+                self.tello.connect()
+                self.is_connected = True
+                self.logMessage.emit("Connected to Tello Drone")
+            except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)